### ISSUE TITLE
Focus gets stuck when navigating through color pickers tools using the keyboard in editor toolbar

### ISSUE THREAD

**Issue by mparvanov** [telerik/kendo-ui-core#5914]
### Bug report

When trying to navigate the editor toolbar with a keyboard, the focus is stuck/not working for  foreColor and backColor color pickers and the user cannot focus the next tool.

### Reproduction of the problem
1. Navigate to https://dojo.telerik.com/aKOKOwiP/7 
1. Focus Kendo Editor toolbar
1. Press Tab (Shift+Tab) several times until focusing the foreColor or backColor tools

### Current behavior
When using Tab(or RightArrow) from the left the focus is stuck on the first colorpicker.
When using Shift+Tab(or LeftArrow) from the right the focus is stuck on the last tool before the colorpicker.

### Expected/desired behavior
The focus shouldn't get stuck and the user could can focus the next available tool using keyboard Tab and Shift+Tab.
https://dojo.telerik.com/aKOKOwiP/10 
### Environment

* **Kendo UI version:** 2020.2.617
The issue is reproducible since R2 2020 (2020.2.513)

Public Item: https://feedback.telerik.com/kendo-jquery-ui/1474743-keyboard-navigation-does-not-work-for-colorpickers
